packages/model-viewer/src/features/scene-graph/sampler.ts

```diff
@@ -146,15 +137,15 @@ export class Sampler extends ThreeDOMElement implements SamplerInterface {
   }
 
   get rotation(): number {
-    return this[$sourceSampler].rotation;
+    return [...this[$threeTextures]][0]['rotation'];
```

Comment:
This feels a touch dangerous - how about copy how it's done in `Image`: `$threeTexture`.

Reply:
Agree, modified this